SimMuon/CSCDigitizer/plugins/CSCChamberMasker.cc

```diff
@@ -181,12 +181,12 @@ CSCChamberMasker::copyDigis(edm::Event & event,
     {
       edm::Handle<C> digis;
       event.getByToken(digiToken, digis);
-      for (typename C::DigiRangeIterator j = digis->begin(); j != digis->end(); j++) 
+      for ( auto j : (*digis) ) 
```

Comment:
shouldn't this be `const auto&` to avoid the copy while preserving the functionality?